### ISSUE TITLE
DINO

### ISSUE THREAD

**Issue by Gh0stW0If** [ChrisMats/medical_transformers#5]
When i only use python classification.py --dino to train on ISIC2019,I got a really poor result.Should i change other parameters in the json file?

**Comment by ChrisMats** [comment]
Hi,
Yes,  you should change the hyperparameters (as described in the paper).

**Comment by Gh0stW0If** [comment]
Thanks,but I use the same hyperparameters settings for lr,wd,EMA.When I pretrain on the ISIC dataset,the weight and bias stop update after about 30 epochs ,train loss and metrics also become stable.  Is this normal？I think training more epochs cannot solve this problem.Did i miss other details?

**Comment by ChrisMats** [comment]
No, this is not normal. The weights and the biases are constantly changing until the end (300 epochs in my case).

**Comment by Gh0stW0If** [comment]
Thanks,I change the optimizer to adamW,and  it goes well.
I still want to try it on DDSM.Where could i get the same format data as yours?
